libraries/Bluefruit52Lib/src/bluefruit.h

```diff
@@ -163,6 +163,8 @@ class AdafruitBluefruit
     uint8_t  connected         (void); // Number of connected
     bool     connected         (uint16_t conn_hdl);
 
+    void getConnectionHandles(uint16_t* connectionHandles, uint8_t maxConnectionHandleCount, uint8_t* actualConnectionHandleCount);
```

Comment:
I would suggest to change the name of the function to getConnectedHandles() and its return type to uint8_t for `actualConnectionHandleCount` instead of pointer.

```C

uint8_t getConnectedHandles(uint16_t* connectionHandles, uint8_t maxConnectionHandleCount);

```

If you are OK with this suggestion, I could help to make the changes/update if you are busy.